app/javascript/test/game.test.js

```diff
@@ -1,46 +1,65 @@
-import React from 'react';
-import axios from 'axios';
-import MockAdapter from 'axios-mock-adapter';
-import { mount } from 'enzyme';
+import React from "react";
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import { mount } from "enzyme";
 
-import './setupTests';
+import Game from "../components/game/Game";
 
-import Game from '../components/game/Game';
-
-describe('Game', () => {
+describe("Game", () => {
   let wrapper;
   let mock;
   beforeEach(() => {
-    wrapper = mount(<Game />)
+    wrapper = mount(<Game />);
     mock = new MockAdapter(axios);
+  });
 
-  })
-
-    test('render a h1 welcome message', () => {
-      expect(wrapper.find("h1").text()).toContain("Tic-Tac-Toe");
-    });
+  test("render a h1 welcome message", () => {
+    expect(wrapper.find("h1").text()).toContain("Tic-Tac-Toe");
+  });
 
-    test('create a new game with post method', () => {
-      mock.onPost('http://localhost:3000/games',  { opponent: "", current_player: '' }).reply(201, {
-          id: "1",
+  test("create a new game with post method", (done) => {
```

Comment:
When you create a new game, you also store the game ID in local storage. It doesn't seem like you're testing this behavior.